src/common/modal/Modal.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import ReactModal from 'react-modal';
```

Comment:
Opening the modal seems to produce error to console
<img width="438" alt="Screenshot 2020-05-07 at 9 12 40" src="https://user-images.githubusercontent.com/1407958/81260660-1e132080-9043-11ea-93e6-cb38b6674399.png">

Reply:
I added the appElement property